pkg/proxy/winkernel/hnsV1.go

```diff
@@ -194,7 +194,7 @@ func (hns hnsV1) getLoadBalancer(endpoints []endpointsInfo, flags loadBalancerFl
 			} else if len(elbPolicy.VIPs) != 0 {
 				continue
 			}
-			LogJson("policyList", plist, "Found existing Hns loadbalancer policy resource", 1)
+			klog.V(1).InfoS("found existing Hns loadbalancer policy resource", "policyList", plist)
```

Comment:
User guide specifies that "policies" is better "policyList".
Can you fix it in other PR cases?